### ISSUE TITLE
move to python 3.5 on win32

### ISSUE THREAD

**Issue by totaam** [Xpra-org/xpra#1072]
*Issue migrated from trac ticket # 1072*

**component:** packaging | **priority:** trivial | **resolution:** fixed | **keywords:** win32

#### 2016-01-05 04:47:17: antoine created the issue

___
> This is only used for sound at present (see #1041) and it is blocked by the lack of released version of cx_freeze for python 3.5: [https://bitbucket.org/anthony_tuininga/cx_freeze/pull-requests/83]. But since trunk supports 3.5, we could always use a known working version (and snapshot it if necessary).
> Low priority.
